src/main/java/com/labsynch/labseer/api/ApiStructureImageController.java

```diff
@@ -60,6 +61,29 @@ public ResponseEntity<byte[]> convertMol(@RequestBody String molStructure,
 
     }
 
+    @RequestMapping(value = "/convertMol/base64", method = RequestMethod.POST)
```

Comment:
This is a simple derivative of the existing `convertMol` route immediately above. All I added was the two lines:
```
Base64.Encoder encoder = Base64.getEncoder();
String encodedString = encoder.encodeToString(image);
```
and changed the return type to `String` instead of `bytes[]`